src/main/java/hudson/plugins/templateproject/ProxyBuilder.java

```diff
@@ -51,11 +54,12 @@ public String getProjectName() {
 
 	@Override
 	public void buildDependencyGraph(AbstractProject project, DependencyGraph graph) {
-		AbstractProject<?, ?> templateProject = (AbstractProject) Hudson.getInstance().getItem(getProjectName());
-		if (templateProject != null) {
-			for (Publisher publisher : templateProject.getPublishersList().toList()) {
-				if (publisher instanceof DependecyDeclarer) {
-					((DependecyDeclarer)publisher).buildDependencyGraph(project, graph);
+		TopLevelItem item = Jenkins.getInstance().getItem(getProjectName());
```

Comment:
This should probably be `getItemByFullName` to account for Folder plugin, shouldn't it?